discovery-provider/alembic/versions/273b8bcef694_user_bank_updates.py

```diff
@@ -0,0 +1,51 @@
+"""user bank updates
+
+Revision ID: 273b8bcef694
+Revises: 436c10e54758
+Create Date: 2021-06-22 17:22:00.102134
+
+"""
+import logging
+from alembic import op
+import sqlalchemy as sa
+from sqlalchemy.orm import sessionmaker
+from solana.publickey import PublicKey
+from src.models import User
+from src.utils.solana import get_address_pair
+
+logger = logging.getLogger(__name__)
+
+# revision identifiers, used by Alembic.
+revision = '273b8bcef694'
+down_revision = '436c10e54758'
+branch_labels = None
+depends_on = None
+
+Session = sessionmaker()
+
+def upgrade():
+    op.create_table('user_bank_txs',
+                sa.Column('signature', sa.String(), nullable=False),
+                sa.Column('slot', sa.Integer(), nullable=False),
+                sa.Column('created_at', sa.DateTime(), nullable=False),
+                sa.PrimaryKeyConstraint('signature')
+                )
+    op.create_table('user_bank_accounts',
+                sa.Column('signature', sa.String(), nullable=False),
+                sa.Column('ethereum_address', sa.String(), nullable=False),
+                sa.Column('created_at', sa.DateTime(), nullable=False),
+                sa.Column('bank_account', sa.String(), nullable=False),
+                sa.PrimaryKeyConstraint('signature')
+                )
+    op.add_column('user_balances',
+                sa.Column('waudio', sa.Integer(), server_default='0', nullable=True)
```

Comment:
Not sure we want integer here - we store balance as a string I believe in other places